.gitignore

```diff
@@ -55,4 +55,29 @@ networkx.egg-info/
 .pytest_cache
 
 # Virtual environment directory
-networkx-dev/
\ No newline at end of file
+networkx-dev/
```

Comment:
This might seem strange, but I'd prefer not to touch `.gitignore` in this PR. I realize building the docs locally dumps these files into the dev environment which can be a bit of an eyesore, but it's an unrelated change that I'd actually propose to handle differently (i.e. with temporary files or `io` objects).

Anyways, let's undo the changes here. If this is something you're interested in pursuing, consider opening a separate PR!